Sources/Valkey/RESP/RESPRenderable.swift

```diff
@@ -14,7 +14,7 @@
 
 import NIOCore
 
-/// A type that can be rendered into a RESP buffer.
+/// A type that the command encoder can render to send as a Valkey command.
```

Comment:
This isn't a full command 
```suggestion
/// A type that the command encoder can render as part of a Valkey command.
```